doc/lib/mqtt.txt

```diff
@@ -59,7 +59,7 @@ In addition to the components above, the MQTT library also depends on C standard
 @subsection mqtt_memory_code Code size
 @brief Code size of the MQTT library.
 
-The following measurements were taken with arm-none-eabi-gcc v9.2.1 (October 2019) with the [MQTT library from January 2020](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/f1a8b698b1d2fd5b8475b11a16d399181c1d867f). These values are rough estimates not tuned for any specific ARM processor.
+The following measurements were taken with arm-none-eabi-gcc v9.2.1 (November 2019) with the [MQTT library from January 2020](https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/f1a8b698b1d2fd5b8475b11a16d399181c1d867f). These values are rough estimates not tuned for any specific ARM processor.
```

Comment:
This was released in November 2019 according to the official page: https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads

Reply:
The build is from October 2019 though.

```
arm-none-eabi-gcc (GNU Tools for Arm Embedded Processors 9-2019-q4-major) 9.2.1 20191025 (release)
```

But I don't think the month mentioned here is important, so I'm ok with the change.